apps/web/components/ServiceWeb/SearchInput/SearchInput.tsx

```diff
@@ -165,7 +167,7 @@ export const SearchInput = ({
                 onSelect(item)
               }}
             >
-              <Text as="span">{item.title}</Text>
+              <span dangerouslySetInnerHTML={{ __html: item.title }} />
```

Comment:
Why is `dangerously` needed for a title?
Does elastic inject `<pre>` or `<em>` into its results?

Reply:
yes - the highlights come as html tags - as they have been thus far ... a few more of these are already in the codebase